modules/oe_content_organisation/oe_content_organisation.post_update.php

```diff
@@ -23,22 +23,27 @@ function oe_content_organisation_post_update_00001(): void {
 }
 
 /**
- * Enable new dependency.
+ * Enable new dependencies.
  */
 function oe_content_organisation_post_update_00002(): void {
-  \Drupal::service('module_installer')->install(['description_list_field']);
+  \Drupal::service('module_installer')->install([
+    'description_list_field',
+    'oe_media',
+    'oe_content_person',
```

Comment:
The module does not depend from `oe_content_person`, the optional `oe_content_organisation_person(_reference)` does.